common/defaults/system_settings.ini

```diff
@@ -8,6 +8,9 @@ usb30_force_enabled = u8!0x0
 ; (note: this is normally not necessary, and ips patches can be used.)
 [ro]
 ease_nro_restriction = u8!0x0
+; Disable screenshot verification
+[capsrv]
+enable_album_screenshot_filedata_verification = u8!0x1
```

Comment:
Are you sure this is correct? 1 sure looks a lot like "yes please do verify screenshots" (and is also the default...)

Reply:
```
capsrv is responsible for validating the MACs for screenshots stored on the SD card, if the setting capsrv!enable_album_screenshot_filedata_verification is non-zero. 
```

- [switchbrew](https://switchbrew.org/wiki/Capture_services#Notes)

Reply:
So if it's non-zero, it verifies MACs, so you need to set it zero.

Reply:
oh, i can read.